libs/hdf-converters/src/sarif-mapper.ts

```diff
@@ -122,25 +118,40 @@ export class SarifMapper extends BaseConverter {
               }
             },
             impact: {path: 'level', transformer: impactMapping},
-            code: '',
+            code: {
+              transformer: (vulnerability: Record<string, unknown>): string => {
+                return JSON.stringify(vulnerability, null, 2);
+              }
+            },
             results: [
               {
                 status: ExecJSON.ControlResultStatus.Failed,
                 code_desc: {
                   path: 'locations[0].physicalLocation',
                   transformer: formatCodeDesc
                 },
-                run_time: 0,
+
                 start_time: ''
               }
             ]
           }
         ],
         sha256: ''
       }
-    ]
+    ],
+    passthrough: {
+      transformer: (data: Record<string, any>): Record<string, unknown> => {
+        const auxData = _.omit(data, ['version']);
+        auxData.runs = auxData.runs.map((run: any) => _.omit(run, ['results']));
```

Comment:
don't use any, esp since we're assuming that `run` is an object/Record<string, unknown> anyways

Reply:
From what I've experienced, typing instances as `unknown` prevents accessing object attributes (which also seem to be the case with this function as well). Is there a workaround or am I just not looking at it right?

Reply:
The primary issue is that `any` is propagated and pollutes the rest of the type system so it's hard to do things safely - any can be anything and so is treated as validly being everything.  If you use unknown instead, you're forced to manually make casts and explicitly specify the "contract" that a given thing is of a specific type.  It is definitely not perfect (everything being precisely typed), but it is significantly better than the alternative of there essentially not being a type at all.